hw/ip/edn/doc/theory_of_operation.md

```diff
@@ -62,38 +70,42 @@ To ensure that the most recently enabled EDN will get next priority for physical
 Once that has happened, the next EDN can be enabled.
 
 If using boot-time request mode, the CSRNG seed material used for the first-activated EDN is the special pre-FIPS seed, which is specifically tested quickly to improve latency.
-The first random values distributed from this EDN will therefore be available roughly 2ms after reset.
+For [Top Earlgrey](../../../top_earlgrey/README.md), the first random values distributed from this EDN are therefore expected to be available roughly 2ms after reset.
+
 The `entropy_src` only creates one pre-FIPS seed, so any other EDNs must wait for their seeds to pass the full FIPS-recommended health checks.
-This means that each subsequent EDN must wait an additional 5ms before it can start distributing data.
+This means that for [Top Earlgrey](../../../top_earlgrey/README.md), each subsequent EDN must wait an additional 5ms before it can start distributing data.
 For instance, if there are three boot-time request mode EDN's in the system, the first will start distributing data 2ms after reset, the second will start distributing data 7ms after reset, and the third will start distributing data 12ms after reset.
 
 ### Auto Request Mode
 
-Before entering auto request mode, it is the responsibility of firmware to first generate an `instantiate` command for the EDN-associated instance via the [`SW_CMD_REQ`](registers.md#sw_cmd_req) register.
-The required `generate` and `reseed` commands must also be custom generated by firmware and loaded into the respective command replay FIFOs via the [`GENERATE_CMD`](registers.md#generate_cmd) and [`RESEED_CMD`](registers.md#reseed_cmd) registers.
+Before entering auto request mode, it is the responsibility of firmware to first load the required `generate` and `reseed` commands into the respective command replay FIFOs via the [`GENERATE_CMD`](registers.md#generate_cmd) and [`RESEED_CMD`](registers.md#reseed_cmd) registers.
```

Comment:
Nit: I wonder whether this could be tweaked slightly. At the moment, it looks a bit like firmware is entering auto request mode, which isn't quite what you mean. The sentence is also quite long. Maybe something like this?

```
Before enabling auto request mode, firmware must configure the EDN's `generate` and `reseed` commands.
These can be loaded into their respective command replay FIFOs through the [`GENERATE_CMD`](registers.md#generate_cmd) and [`RESEED_CMD`](registers.md#reseed_cmd) registers.
```

Reply:
This makes sense, thanks @rswarbrick .